src/main/java/seedu/sudowudo/logic/commands/EditCommand.java

```diff
@@ -87,31 +87,35 @@ public CommandResult execute() {
         previousTemplate = itemToModify.deepCopy();
 
         for (String[] editField : editFields) {
-            switch (editField[0]) {
-                case "desc":
-                case "description":
-                    model.setItemDesc(itemToModify, editField[1]);
-                    break;
-                case "start":
-                    model.setItemStart(itemToModify,
-                            new DateTimeParser(editField[1])
-                                    .extractStartDate());
-                    break;
-                case "end":
-                case "by":
-                    model.setItemEnd(itemToModify,
-                            new DateTimeParser(editField[1]).extractStartDate());
-                    break;
-                case "period":
-                    DateTimeParser parser = new DateTimeParser(editField[1]);
-                    model.setItemStart(itemToModify, parser.extractStartDate());
-                    model.setItemEnd(itemToModify, parser.extractEndDate());
-                    break;
-                default:
-                    // field names not valid
-                    return new CommandResult(MESSAGE_INVALID_FIELD);
+            try {
+                switch (editField[0]) {
+                    case "desc":
```

Comment:
Code standard violation. `case` statement should be flush to `switch`.

Should look like:
```java
switch(editField[0]) {
case "desc":
case "description":
    // statements blah blah
    break;
case "start":
    // statements blah blah blah
    break;
// ...
```

I fixed this in #104 though so just leave it